src/devices/Pcd8544/NokiaFont.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// The specific PCD8544 font for Nokia 5110
+    /// </summary>
+    public static class NokiaCharacters
+    {
+        /// <summary>
+        /// ASCII Font specific to the PCD8544 Nokia 5110 screen
+        /// </summary>
+        public static readonly byte[][] Ascii = new byte[][]
+        {
+        new byte[] { 0x00, 0x00, 0x00, 0x00, 0x00 }, // 20
```

Comment:
where do these come from? are those columns in binary?

Reply:
Yes, adding